src/Terminal/commands/expr.ts

```diff
@@ -17,7 +17,7 @@ export function expr(
   const expr = args.join("");
 
   // Sanitize the math expression
-  const sanitizedExpr = expr.replace(/s+/g, "").replace(/[^-()\d/*+.]/g, "");
+  const sanitizedExpr = expr.replace(/s+/g, "").replace(/[^-()\d*/+.\%]/g, "");
```

Comment:
When you remove the unnecessary escape and reordering that doesn't change anything as well, you and up with:

```suggestion
  const sanitizedExpr = expr.replace(/s+/g, "").replace(/[^-()\d/*+.%]/g, "");
```
Which is just adding `%`